### ISSUE TITLE
[Feature Request] Better Repository statistics

### ISSUE THREAD

**Issue by fihorvat** [ForkIssues/TrackerWin#1554]
Hello, first and foremost I really love your application so why don't we make it even better.

Currently, repository statistics are only based on commits.
Commits can sometimes be very inaccurate since we can squash the commits to make git history easier to read. The "issue" is that a quick bugfix with 1 line of code is treated the same as 1 squashed commit with xyz numbers of changes (added, deleted, modified lines). 
So some large branches where we refactor complete applications and have been working on them for weeks are treated as 1 contribution.

Could we add detailed statistics which are based on changed lines of code 
so something like this: [how-to-count-total-lines-changed](https://stackoverflow.com/questions/1265040/how-to-count-total-lines-changed-by-a-specific-author-in-a-git-repository)
This is also great: `git ls-files | xargs -n1 git blame --line-porcelain | sed -n 's/^author //p' | sort -f | uniq -ic | sort -nr`

We could move this to a new tab or under the current statistics.

**Comment by DanPristupov** [comment]
> This is also great: `git ls-files | xargs -n1 git blame --line-porcelain | sed -n 's/^author //p' | sort -f | uniq -ic | sort -nr`

It took 35 seconds on my computer! The one which Fork uses takes milliseconds.

**Comment by fihorvat** [comment]
Yes, I know this is an expensive calculation, but I'm not saying we should use this command. It's just an example. This feature will likely need some processing in the background and results will not be shown immediately.
There are many tools that do this, so maybe it would be a cool feature and certainly the only git management tool that has such detailed statistics.
[git-quick-stats](https://github.com/arzzen/git-quick-stats)
[git-fame](https://github.com/casperdcl/git-fame)
